src/main/java/it/gov/pagopa/pu/workflow/wf/classification/iuf/wfclassification/IufClassificationWFImpl.java

```diff
@@ -89,15 +95,20 @@ public void notifyPaymentsReporting(IufClassificationNotifyPaymentsReportingSign
     Long clearedResult = clearClassifyIufActivity.deleteClassificationByIuf(
       signalDTO.getOrganizationId(),
       signalDTO.getIuf());
-
     log.info("IUF receipt classification cleared cleared {} records for {}", clearedResult, signalDTO);
-    List<Transfer2ClassifyDTO> transfer2ClassifyDTOList = signalDTO.getTransfers().stream()
-      .map(transfer -> Transfer2ClassifyDTO.builder()
+
+    List<Transfer2ClassifyDTO> transfer2ClassifyDTOList = new ArrayList<>();
```

Comment:
Stream reduce operation will collect elements in an efficient way... Thus, it could be more useful not remove it.
Anyway, it's not a problem, because we can use the size of the input list di efficiently build a List ready to contains the entire required data
```suggestion
    List<Transfer2ClassifyDTO> transfer2ClassifyDTOList = new ArrayList<>(signalDTO.getTransfers().size());
```